e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -107,6 +107,106 @@
               "createOptions": ""
             }
           },
+          "DirectMethodSender": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "InvocationSource": {
+                "value": "Cloud"
+              },
+              "ServiceClientConnectionString": {
+                "value": "<ServiceClientConnectionString>"
+              },
+              "TargetModuleId": {
+                "value": "DirectMethodReceiver"
+              },
+              "TransportType" : {
+                "value": "Mqtt"
```

Comment:
1 pair of DM sender and receiver uses amqp and the other one uses mqtt.  Please set transport type for DirectMethodSender and DirectMethodReceiver to Amqp.